src/wikidata/getItemsLabels.js

```diff
@@ -0,0 +1,25 @@
+import { DEFAULT_LANGS_CODES } from "../constants/langs";
+import addLabel from "../lib/addLabel";
+import getEntitiesFromWikidata from "./getEntitiesFromWikidata";
+
+export default async function getItemsLabels(ids, languageCode) {
```

Comment:
getItemLabels, or getLabelsByItemList sounds better than a double plural, but it's fine

Reply:
true true, the intention is to surface the fact that you are using ids array, and then you get its label, so should actually be `getItemsLabel` according to what I just said 😂

Reply:
then I would keep `getLabelsByItemList` for getting something like 
```
Label {
  id: 1
  items: [1,3,4]
}

SELECT * FROM Label WHERE items IN [1,3]
```